cmd/agent/flags.go

```diff
@@ -108,7 +108,50 @@ var flags = []cli.Flag{
 		Value:   "auto-detect",
 	},
 
-	// TODO: add flags of backends
+	// backend docker
```

Comment:
can these flags be moved to engine (function in interface that would return them)? And than append them by looping all engines and calling that func?

Reply:
I had a similar idea what if we could move backend flag declaration into pipeline/backend/x/flags.go ...

but if we want to have a simple codebase for now we have to do it as is ... and I might come up with an elegant idea later :)

Reply:
Do we need to know which backend is active / selected to get those flags. Couldn't we just get a list of all available options from each backend and add them to the list.

Reply:
if we just import things ...
we get:
 - import circles and
 - https://woodpecker-ci.org/docs/next/development/architecture get violated